package.json

```diff
@@ -4,16 +4,16 @@
   "description": "",
   "scripts": {
     "postinstall": "tsd reinstall --overwrite --save",
-    "start": "webpack-dev-server --port 888"
+    "start": "webpack-dev-server --port 6767"
   },
   "author": "Pablo Núñez",
   "license": "ISC",
   "dependencies": {
     "bootstrap": "^3.3.5",
     "jquery": "^2.1.4",
-    "office-ui-fabric-react": "^0.22.1",
+    "office-ui-fabric-react": "^0.27.0",
     "react": "^15.1.0",
```

Comment:
The only reason you need react as an NPM dependency is if you need webpack to resolve it for bundling. TypeScript only cares about the d.ts. I marked react as an external in your project in the webpack.config so that it reduces the bundle size, but also removes accidentally duping react in the bundle. I ran into an issue with npm2 where webpack wants to bundle your app's react dependency along with fabric-react's react dependency. Pretty confusing, not sure why, but it causes bugs because two copies of react are on the page. But best practice anyway is to keep static libs separate from app code.